src/api/app/models/notifications/rss_feed_item.rb

```diff
@@ -20,6 +20,18 @@ def self.cleanup
       end
     end
   end
+
+  def title
+    event.subject
+  end
+
+  def description
+    ApplicationController.renderer.new.render(
```

Comment:
For reusing the same views as en the EventMailer instead of having to calculate it somewhere out of the model...